src/hooks/useFirstMountState.ts

```diff
@@ -0,0 +1,13 @@
+import { useRef } from "react";
+
+export function useFirstMountState(): boolean {
```

Comment:
dublicate

Reply:
`hooks.ts` was removed in favor of separate hook-per-file